gateway-service-knoxtoken/src/test/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceResourceTest.java

```diff
@@ -1006,7 +1007,7 @@ public void testTokenLimitChangeAfterAlreadyHavingTokens() throws Exception {
     for (int i = 0; i < numberOfPreExistingTokens; i++) {
       tr.doGet();
     }
-    Response getKnoxTokensResponse = tr.getUserTokens(USER_NAME);
+    Response getKnoxTokensResponse = tr.getUserTokens(USER_NAME, null, null);
```

Comment:
Should these additional parameters be required here? I would expect the same result from a call to `tr.getUserTokens(USER_NAME);` Is this a backward-compatabiltiy concern?

Reply:
Let me add overloaded methods to support multiple cases.

Reply:
Fixed.